test/system/moderation_tests.rb

```diff
@@ -0,0 +1,75 @@
+require "application_system_test_case"
+
+class ModerationTest < ApplicationSystemTestCase
+
+  def setup
+    visit "/"
+    click_on "Login"
+    fill_in 'user_session[username]', with: 'palpatine'
+    fill_in 'user_session[password]', with: 'secretive'
+    click_on "Log in"
+  end
+
+  test "banning and unbanning a user" do
+    visit "/profile/#{users(:bob).username}"
+
+    find("a#info-ellipsis").click
+    accept_confirm "Are you sure? The user will no longer be able to log in or publish, and their content will be hidden." do
+      find("a[href='/ban/#{users(:bob).id}'").click()
```

Comment:
Is there a particular reason for inconsistency?
On line 16, you call `.click` and on line 18 you call `.click()`, similar on other lines also.

Reply:
Haha, I will this minor problem :+1:. What do you think is better?

Reply:
In my home opinion, since most of the programmers come from C based languages, the `.click()` is probably more readable and indicates that it is a function call. What do you think?

Reply:
Yeah that's what I did :+1: